client/src/pages/EditProfilePage/EditProfilePage.tsx

```diff
@@ -21,32 +22,167 @@ const EditProfilePage = () => {
   const fileInputRef = useRef<HTMLInputElement>(null);
 
   const [formData, setFormData] = useState({
+    cohort: "",
```

Comment:
super nitpicky but could we order these the same as they appear on the form?

Reply:
Absolutely, good call!